source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md

```diff
@@ -0,0 +1,377 @@
+
+## Prepare the Local Environment
+
+<Partial file="drupal-9/prepare-local-environment.md" />
+
+Create a new folder to use while working on the migration. This folder will contain two subdirectories that you'll create in the next sections, one for the site on the former platform, and one for the Pantheon site.
+
+This doc uses the following aliases:
+
+- **Alias:** `SITE`
+- **Site Name:** `anita-drupal`
+- **Working folder** ?
+- **Old site folder** `FORMER-PLATFORM`
+- **Pantheon site folder** `PANTHEON-D9`
+
+### Create a Local Copy of the Old Site's Code
+
+1. Obtain a local copy of your old site's code.  Your **code** is all custom and contributed modules or plugins, themes, and libraries. The codebase should not include the `sites/default/files` directory, or any other static assets you do not want tracked by version control.
+
+1. Export the database and media files (`sites/default/files`) from the old platform, but do not add them or upload any files to Pantheon.
+
+### Retrieve a Local Copy of the Pantheon Site's Code
+
+1. From the **<span class="glyphicons glyphicons-wrench"></span> Dev** environment of the Site Dashboard, set the site's Development Mode to Git:
+
+  ![Git connection mode](../../../images/dashboard/connection-mode-git.png)
+
+1. Copy the `git clone` command for the site repository.
+
+  The command should look similar to the following:
+
+  ```bash
+  git clone ssh://codeserver.dev.{site-id}@codeserver.dev.{site-id}.drush.in:2222/~/repository.git
+  ```
+
+1. Run the `git clone` command inside your working folder
+
+<!--
+### Site Structure
+
+<Partial file="ic-upstream-structure.md" />
+-->
+
+## Add in the Custom and Contrib Code Needed to Run Your Site
+
+What makes your site code unique is your selection of contributed modules and themes, and any custom modules or themes your development team has created. These customizations need to be replicated in your new project structure.
+
+### Composer packages
+
+1. Copy your package list from the "requires" section of your site's composer.json and add it to the new site's composer.json. If your composer.json defines additional repositories or patches, copy those over too. Take care not to overwrite the `upstream-configuration` package & repository.
+
+  Note: if your old site has custom patches in its codebase, make sure to copy those over as well.
+
+1. `composer update`
+
+1. `git status` to see if files have been added that aren't ignored by `.gitignore`. If anything shows up other than `composer.*`, add it to `.gitignore` until `git status` only shows the composer files being modified.
+
+1. `git add composer.*; git commit -m "Add composer packages"`
+
+  <Accordion title="Repositories and patches in composer.json" id="repositories-and-patches-in-composer-json" icon="info-sign">
+
+  Repository in old site's `composer.json`
+
+  ![repository in old composer.json](https://i.imgur.com/hO0snBW.png)
```

Comment:
convert to codeblock

Reply:
These screenshots will need to be re-done with pantheon-systems/drupal-recommended

Reply:
@namespacebrian I'm removing these screenshots for now - I can't think of a good way to explain what they're showing that's both easy to understand and easy to understand that it's for display purposes only. We can come back to this in future iterations